docs/brand.md

```diff
@@ -132,7 +132,3 @@ Illustration is most effective when it is precise in choosing a single message t
   <img src="{{ site.baseurl }}/assets/img/illustrations/fail.svg" role="img" alt="Failure illustration" class="display-block grid-col flex-auto flex-align-center">
   <img src="{{ site.baseurl }}/assets/img/illustrations/id-confirm.svg" role="img" alt="Confirmed identification illustration" class="display-block grid-col flex-auto flex-align-center">
 </div>
-
-## Slide decks
```

Comment:
Per failing build: Need to remove the `sidenav` entry for Slide decks from frontmatter at the top of the file.

Reply:
Fixed here 86e76fb